spec/models/conference_spec.rb

```diff
@@ -778,6 +778,35 @@
     end
   end
 
+  describe '#scheduled_event_distribution' do
+
+    before(:each) do
+      @conference = create(:conference)
```

Comment:
OK, but should we use this as then we will have to change its start and end date so that the validations would pass and that's why I see most of the test are creating another conference variable instead of using it.